RomeALincei/ANLcr15.xml

```diff
@@ -0,0 +1,854 @@
+                                <incipit xml:lang="gez">ተአምሪሁ፡ <gap reason="ellipsis"/> ወኮነ<supplied reason="omitted">፡</supplied>
+                                    በአሐቲ፡ እመዋዕል፡ እንዘ<supplied reason="omitted">፡</supplied> ሀለው፡
+                                    ጉባዔ፡ ሕዝብ፡ መሲሐውያን፡ ምስለ፡ አባ፡ በርሱማ፡ ዕሩቅ፡</incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2.59">
+                                <locus from="76ra" to="77ra"/>
+                                <title type="complete" ref="LIT6629MGHealing"/>
+                                <incipit xml:lang="gez">ተአምሪሁ፡ <gap reason="ellipsis"/> <supplied reason="omitted">ወነበረት፡ አሐቲ፡ ብእሲት፡ ጽንስት፡</supplied> 
+                                    በፍቅሩ፡ ለጊዮርጊስ<supplied reason="omitted">፡</supplied> ዘአምነት፡ ብእሲት፡
+                                    እምአዋልደ፡ እስራኤል፡ <cb n="b"/> ወዐቢይ፡ ፍጥረታ፡ በውስተ፡ ሕዝብ፡ ወሰሚዓ፡ ከመ፡
+                                    ይገበር፡ ተአምረ፡ ዓቢየ፡ አምነት፡ ወትቤ፡</incipit>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2.60">
+                                <locus from="77rb" to="77va"/>
+                                <title type="complete" ref="LIT7408MGWidow"/>
+                                <incipit xml:lang="gez">ተአምሪሁ፡ <gap reason="ellipsis"/> ወሀለወት፡ አሐቲ፡ መበለት<supplied reason="omitted">፡</supplied>
+                                    ዘትነብር፡ ጐረ፡ ቤተ፡ ክርስቲያኑ፡ ለቅዱስ፡ ጊዮርጊ<supplied reason="omitted">ስ፡</supplied> 
+                                    ወመሰጠ፡ ጊዮርጊ<supplied reason="omitted">ስ</supplied>፡ አንቄ፡ ዶርሆሃ፡
+                                    ወትቤ፡</incipit>
+                            </msItem>
+                        </msItem>
+                        </msItem>
+                      
+                    </msContents>
+                    
+             <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf" quantity="79">79</measure>
+                                    <note>The number of folios is not indicated in the catalogue,
+                                        but can be deduced from the description.</note>
+                                    <dimensions type="outer" unit="mm">
+                                        <height unit="mm">260</height>
+                                        <width unit="mm">170</width>
+                                    </dimensions>
+                                </extent>
+                                <collation>
+                                    <note><locus target="#1"/> and <locus from="78" to="79"/> are
+                                        guard leaves.</note>
```

Comment:
If I am not confusing things, based on Eliana´s terminology, Eugenia told me to use "endleaves" instead of guard leaves.